crates/telio-firewall/src/firewall.rs

```diff
@@ -310,6 +319,12 @@ struct TcpConn {
     pubkey: PublicKey,
 }
 
+#[derive(Debug, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
+struct UdpConn {
```

Comment:
Maybe it makes sense to merge `UdpConn` and `TcpConn` to one struct, as both seems to be identical by now :).

Reply:
Yeaaaah, however, it may not last long :). I kind of prefer keeping things type-safe